src/test/scala/nl/knaw/dans/easy/solr/ConfSpec.scala

```diff
@@ -0,0 +1,57 @@
+package nl.knaw.dans.easy.solr
+
+import java.io.ByteArrayOutputStream
+
+import org.scalatest.{FlatSpec, Matchers}
+
+import scala.runtime.BoxedUnit
+
+class ConfSpec extends FlatSpec with Matchers {
+
+  "printHelp" should "produce help info with values from custom properties file" in {
```

Comment:
Is it useful to test this? It is really Scallop-functionality and not our own logic that you are testing here.